Stripe/Checkout/STPCheckoutOptions.m

```diff
@@ -67,6 +67,23 @@ - (NSString *)stringifiedJSONRepresentation {
     return [[NSString alloc] initWithData:[NSJSONSerialization dataWithJSONObject:values options:0 error:nil] encoding:NSUTF8StringEncoding];
 }
 
+- (void)setLogoImage:(STP_IMAGE_CLASS * __stp_nullable)logoImage {
```

Comment:
I don't see this method being called in any of the example code, but it seems necessary in order for the Dribbble logo to appear. Could you explain the usage of this method?